### ISSUE TITLE
improve docs by writing chemical species upright

### ISSUE THREAD

**Issue by Mo-Gul** [zdroid/ChemEquations.jl#9]
Currently 

https://github.com/zlatanvasovic/ChemEquations.jl/blob/393e520d3e586e93596291519d10f51e1038878f/docs/src/index.md?plain=1#L7-L9

is rendered as

![grafik](https://user-images.githubusercontent.com/17992976/155931693-bb4a624f-21b8-4662-b4a7-9c613ea46169.png)

on <https://zlatanvasovic.github.io/ChemEquations.jl/dev/>.

But as you for sure know, chemical species should be written in an upright font. I think it should be possible to achieve that with the `mhchem` extensions of `MathJax` and `KaTeX`, depending on which renderer you are using. Please have a look at

<https://mhchem.github.io/MathJax-mhchem/>

**Comment by zlatanvasovic** [comment]
You're absolutely right, I'll check your proposal. If you manage to make a working solution, you're always welcome to submit a PR here. :smile:

**Comment by Mo-Gul** [comment]
I'd love to provide a PR, but unfortunately I am an absolute newbie to Julia and thus have no clue how I would get the resulting HTML output from `Documenter.jl`. Said that I can't _test_ if a proposal works ...

**Comment by longemen3000** [comment]
the text encased in math quotes is valid latex. you could write straight text by using `\text{Text}` or `\textrm{Text}`. for example:
```
 \text{CH}_4 + \text{O}_2 \to \text{CO}_2 + \text{H}_2 \text{O} 
```
renders as:
![image](https://user-images.githubusercontent.com/38795484/156269204-6130699b-c2fc-4dc0-bd03-9ab7a6ec1102.png)

**Comment by ProfLeao** [comment]
Has anything been done in this direction? If not, I can try to do something.

**Comment by Mo-Gul** [comment]
Nope, the equation is still rendered as "pure math". Of course the trivial fix would be to apply longmen's solution (assuming it works). Would you be willing to have a look at `mhchem` or what is your plan?

**Comment by ProfLeao** [comment]
I plan to enable MathJax3 as the default math engine and then use mhchem. I believe this will provide a greater number of possibilities for writing chemical equations in the documentation.

**Comment by Mo-Gul** [comment]
That sounds great to me, but let's hope that @zlatanvasovic is still active and will approve your PR.
I keep my fingers crossed ...

**Comment by ProfLeao** [comment]
I've written him a few emails. In his own time he will. 

But this is a little more complicated, the intervention in the documentation is a bit deeper. The language has changed a lot since the package was last updated. 

Take the expected behavior of the `equationmatrix` method for example, it's not like that anymore.

```
jldoctest
julia> equationmatrix(ce"H2 + Cl2 → HCl")
2×3 Array{Int64,2}:
 2  0  1
 0  2  1
```

I'm currently tracking this kind of update, otherwise the build won't work.

**Comment by ProfLeao** [comment]
PR https://github.com/zlatanvasovic/ChemEquations.jl/pull/11 to see the proposed updates.

**Comment by ProfLeao** [comment]
https://github.com/zlatanvasovic/ChemEquations.jl/pull/11#issuecomment-2229792471
